web/embed-dart.html

```diff
@@ -42,14 +42,14 @@
 
   <body class="fullbleed vertical layout" unresolved>
     <div class="flex layout horizontal">
-      <div id="leftPanel" class="relative vertical layout flex-6">
+      <div id="leftPanel" class="relative vertical layout">
         <dartpad-buttons></dartpad-buttons>
-        <dartpad-editor id='topPanel' dart class="flex-5 layout vertical"></dartpad-editor>
-        <horizontal-splitter class="splitter"></horizontal-splitter>
-        <dartpad-console id='bottomPanel' class="flex-2 layout vertical"></dartpad-console>
+        <dartpad-editor id='topPanel' dart class="layout vertical"></dartpad-editor>
+        <horizontal-splitter class="splitter" horizontal></horizontal-splitter>
```

Comment:
I just want to confirm that this is intended to be a `horizontal` attribute and not a `horizontal` class.